frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -168,13 +168,13 @@ export class CatalogTileViewPage extends React.Component {
       return null;
     }
 
-    const { obj, tileName, tileImgUrl, tileIconClass, tileProvider, tileDescription } = item;
+    const { obj, tileName, tileImgUrl, tileIconClass, tileProvider, tileDescription, kind } = item;
     const uid = obj.metadata.uid;
     const iconClass = tileIconClass ? normalizeIconClass(tileIconClass) : null;
     const vendor = tileProvider ? `provided by ${tileProvider}` : null;
     return (
       <CatalogTile
-        id={uid}
+        id={`${kind}-${obj.metadata.name}`}
```

Comment:
Is this guaranteed to be unique if we're showing resources from more than one namespace? We might do that for image streams.

Reply:
wouldn't we then be showing duplicate tiles?

Reply:
ImageStreams are single namespace (openshift). But CSVs are namespaced...

Reply:
We want to eventually show image streams from both `openshift` and the current namespace.

Reply:
> wouldn't we then be showing duplicate tiles?

Potentially. We'd need a way to differentiate content from different namespaces that have the same name. We have this problem already to some degree since display names are not unique.

Reply:
ok, I'll see what I can do to guarantee uniqueness

Reply:
Changed to use `data-test` rather than `id` so non-unique won't cause runtime problems if they do occur.